application-tour-ui/src/main/resources/TourCode/StepSheet.xml

```diff
@@ -238,6 +239,7 @@ require(['jquery'], function ($) {
                   targetPage  : (targetPage? targetPage: ''),
                   targetAction: (targetAction? targetAction: ''),
                   queryString : (queryString? queryString: ''),
+                  reflex      : (reflex? reflex: 0),
```

Comment:
I don't understand this. Why not use the value of ``reflex`` as is since it's a boolean (returned by ``is(':checked')``).